aidants_connect_web/admin.py

```diff
@@ -22,38 +23,71 @@
 admin_site = OTPAdminSite(OTPAdminSite.name)
 
 
-class AidantAdmin(DjangoUserAdmin):
-    # The forms to add and change aidant instances
+class VisibleToStaff(ModelAdmin):
+    """A mixin to make a ModelAdmin visible to staff users."""
+
+    def has_module_permission(self, request):
+        return request.user.is_staff
+
+    def has_view_permission(self, request, obj=None):
+        return self.has_module_permission(request)
+
+    def has_add_permission(self, request, obj=None):
+        return self.has_module_permission(request)
+
+    def has_change_permission(self, request, obj=None):
+        return self.has_module_permission(request)
+
+    def has_delete_permission(self, request, obj=None):
+        return self.has_module_permission(request)
+
+
+class StaffStaticDeviceAdmin(VisibleToStaff, StaticDeviceAdmin):
```

Comment:
Que penses-tu d'essayer d'exprimer l'idée d'"admin métier", par exemple, "StaffAdmin" 
```suggestion
class StaticDeviceStaffAdmin(VisibleToStaff, StaticDeviceAdmin):
```